### PR TITLE
Remove unused prefixes

### PR DESCRIPTION
**Pull request by ejnan**
Remove phone, email and addresses from original address book

### CODE REVIEW

File: src/main/java/seedu/address/logic/Messages.java [hunk 1]
```diff
@@ -37,16 +37,15 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
...
-                .append("\n Admitted: ")
-                .append(person.getAdmissionDate())
-                .append("\n Ward: ")
+                .append("; Ward No.: ")
                 .append(person.getWard())
-                .append("\n Tags: ")
-                .append(person.getTags());
+                .append("; Admission Date: ")
+                .append(person.getAdmissionDate())
+                .append("; Tags: ");
```
Comment (iamtr) [inline src/main/java/seedu/address/logic/Messages.java]: Just curious, why was this changed? Is it due to the fact that we can fit everything in the screen?

**Review by iamtr — COMMENTED** [review]
Other items look good to me, just curious on the changes in Messages.java

File: src/main/java/seedu/address/logic/Messages.java (hunk 1 above)
Comment (ejnan) [reply src/main/java/seedu/address/logic/Messages.java]: thanks for pointing this out, I just changed it back

File: src/main/java/seedu/address/logic/Messages.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
     }
 
     /**
-     * Formats the {@code person} for display to the user.
+     * Formats the {@code persgeton} for display to the user.
```
Comment (swtan346) [inline src/main/java/seedu/address/logic/Messages.java:35]: I think there's a typo here

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 3]
```diff
@@ -39,20 +33,17 @@ public class AddCommandParser implements Parser<AddCommand> {
...
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_ADDRESS, PREFIX_PHONE, PREFIX_EMAIL,
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME,
                 PREFIX_DOB, PREFIX_IC, PREFIX_ADMISSION_DATE, PREFIX_WARD)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
         }
 
-        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS);
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_IC);
```
Comment (swtan346) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:45]: There should only be 1 date of birth right?
For admission date and ward, should we limit to 1 as well?

**Review by swtan346 — CHANGES_REQUESTED** [review]
LGTM, just some minor issues
